app/src/main/res/values/strings.xml

```diff
@@ -554,4 +554,6 @@ Upload your first media by tapping on the add button.</string>
   <string name="share_text">Upload photos to Wikimedia Commons on your phone Download the Commons app: %1$s</string>
   <string name="share_via">Share app via...</string>
   <string name="image_info">Image Info</string>
+  <string name="dialog_box_text_first">Why should</string>
```

Comment:
Please pass media.getDisplayTitle as parameter into string instead of first and second string. If you don't know how to do an example exists at line 554: <string name="share_text">Upload photos to Wikimedia Commons on your phone Download the Commons app: %1$s</string> or here https://stackoverflow.com/questions/2397613/are-parameters-in-strings-xml-possible

Reply:
@neslihanturan the change is done and thanks for the help .